CoEpi/repo/CoepiRepo.swift

```diff
@@ -91,11 +91,15 @@ class CoEpiRepoImpl: CoEpiRepo {
 
     // TODO clarify with Rust lib, does it store the keys or we pass them
     func sendReport(report: CenReport) -> Completable {
-        if let lastCenKey = cenKeyDao.getLatestCENKey() { // TODO last n keys?
+        switch cenKeyDao.generateAndStoreCENKey() {  // TODO last n keys?
+        case .success(let key):
             // TODO clarify id
-            return api.postCenReport(cenReport: MyCenReport(id: "123", report: report, keys: lastCenKey.cenKey))
-        } else {
-            return Completable.error(RepoError.userHasNoCenKeys)
+            return api.postCenReport(cenReport: MyCenReport(id: "123", report: report, keys: key.cenKey))
```

Comment:
@i-schuetz We can generate random report id:
```
let id = UUID().uuidString
```

Reply:
We should ask the backend, not sure what they expect there.